### PR TITLE
Add Position:isWalkable()

### PR DESCRIPTION
**Pull request by Riverlance**
This function Position:isWalkable() is very useful for be used.

@WibbenZ

### CODE REVIEW

**Comment by Kamenuvol** [comment]
Tile is walkable or not, not position

**Comment by Riverlance** [comment]
Ooh... I see. How can I fix this in here. Let me see.

File: data/lib/core/position.lua [hunk 1]
```diff
@@ -18,29 +18,29 @@ function Position:getNextPosition(direction, steps)
...
 end
 
-function Position:moveUpstairs()
-	local isWalkable = function (position)
-		local tile = Tile(position)
-		if not tile then
-			return false
-		end
+function Position:isWalkable()
+	local tile = Tile(position)
```
Comment (WibbenZ) [inline data/lib/core/position.lua]: Remove this line, use Tile:isWalkable() on line 21.
Then replace tile with self on all functions

ex (line 23)
```
if not self then
...
end
```

Comment (Riverlance) [inline data/lib/core/position.lua]: How can I cancel this pull request? I will do another.

Comment (WibbenZ) [inline data/lib/core/position.lua]: You can probbly do changes here: https://github.com/Riverlance/forgottenserver/tree/patch-1
And they should be updated here when you push things there...

Option 2 install the git programs and hope for the best (that is what I do everytime, works about 70% of the times haha)

Option 3, below the comment thingi you should have an option to close the PR.

**Comment by Riverlance** [comment]
@WibbenZ How can I edit 2 or more files and do only 1 pull request on GitHub?

**Comment by WibbenZ** [comment]
As I said, use this link and edit the files there: https://github.com/Riverlance/forgottenserver/tree/patch-1
But you deleted the branch so idk if you can do it unless you restore the branch and open this PR again.

**Pull request closed without merging**